tools/slurm/inference_joint_metadata.slurm.sh

```
@@ -0,0 +1,53 @@
+echo "Job ID: $SLURM_JOB_ID"
+echo "Node:   $SLURM_NODELIST"
+echo "Date:   $(date)"
+
+PREPRO_DIR=/mnt/data2/nkubrakov/joint/preprocessing_scaffold_v1
+CHECKPOINT_DIR=/mnt/data2/nkubrakov/experiments_3_dataset/training/metadata_adduct_ce_ionmode_attn
+OUTPUT_DIR=/mnt/data2/nkubrakov/experiments_3_dataset/metadata_adduct_ce_ionmode_attn
+
+mkdir -p "${OUTPUT_DIR}"
+mkdir -p /home/nkubrakov/simba-integration/logs
+
+cd /home/nkubrakov/simba-integration
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)